security/oc_tls.c

```diff
@@ -1630,43 +1667,26 @@ verify_certificate(void *opq, mbedtls_x509_crt *crt, int depth, uint32_t *flags)
     }
   }
   OC_DBG("verified certificate at depth %d", depth);
-  if ((oc_pki_get_verify_certificate_cb()(peer, crt, depth, flags) != 0) ||
```

Comment:
yes it could be used.

`oc_pki_get_verify_certificate_cb ` is used in https://github.com/iotivity/iotivity-lite/pull/397/files#diff-de1feb9ee6daaad5a8bf3d46497c0a8396e119a93f70734dff4d741dc988222eR1522